src/server/Document/component.jsx

```diff
@@ -14,14 +14,20 @@ const Document = ({
   data,
   styleTags,
   helmet,
+  service,
   isAmp,
 }) => {
   const htmlAttrs = helmet.htmlAttributes.toComponent();
   const meta = helmet.meta.toComponent();
   const title = helmet.title.toComponent();
   const links = helmet.link.toComponent();
   const headScript = helmet.script.toComponent();
-  const serialisedData = JSON.stringify(data);
+  const clientDataObj = {
```

Comment:
We should not need to do this. the client.js files currently uses the service prop on this global object. We should refactor client.js to no longer require this. https://github.com/bbc/simorgh/issues/1679